kernel/kernel-config-server/README.md

```diff
@@ -157,3 +176,26 @@ management.endpoints.web.exposure.include=refresh
 spring.cloud.config.server.health.enabled=false
 
 ```
+
+**cloud config supported for git type repository**
+
+```
+SPRING_CLOUD_CONFIG_SERVER_COMPOSITE_0_URI=<your-git-repository-URL>
+SPRING_CLOUD_CONFIG_SERVER_COMPOSITE_0_TYPE=git
+SPRING_CLOUD_CONFIG_SERVER_COMPOSITE_0_DEFAULT_LABEL=<your-git-repository-branch>
+```
+
+**cloud config supported for svn type repository**
+
+```
+SPRING_CLOUD_CONFIG_SERVER_COMPOSITE_0_URI=<your-svn-repository-URL>
+SPRING_CLOUD_CONFIG_SERVER_COMPOSITE_0_TYPE=git
```

Comment:
for svn, why is type git ? Should we even support svn ?

Reply:
I wanted to give example that we can support different repository type with same config server.
If uri is svn then type will be svn. I will correct it.
Or should I not mention about svn here?

Reply:
lets not mention svn